_capabilities/analysis-showback.md

```diff
@@ -12,38 +12,71 @@ order: 2
 
 ---
 
-# Data Analysis and Showback
+# Data Visualization and Showback
 
-
-{% include domains-loop.md %}
```

Comment:
@vnm-fodo @drumsnob We shouldn't remove this module but consider just including all the Domains as that helps other users "connect the dots."

Reply:
@drumsnob Let's add the module back the next time you do your editing pass. We can adjust it once we accept your changes so that the Domains module reflects all the appropriate Domains. Thanks!